server/cli/accounting.go

```diff
@@ -48,37 +48,39 @@ func runGetAcct(cmd *cobra.Command, args []string) {
 		cmd.Usage()
 		return
 	}
-	server.RunGetAcct(Context, args[0])
+	client := client.NewAdminClient(&Context.Context, Context.Addr, client.Accounting)
+	body, err := client.GetYAML(args[0])
+	if err != nil {
+		log.Error(err)
+		return
+	}
+	fmt.Printf("Accounting config for prefix %q:\n%s\n", args[0], body)
 }
 
 // A lsAcctsCmd command displays a list of acct configs by prefix.
 var lsAcctsCmd = &cobra.Command{
-	Use:   "ls [options] [key-regexp]",
+	Use:   "ls [options]",
 	Short: "list all accounting configs by key prefix",
 	Long: `
-List accounting configs. If a regular expression is given, the results of
```

Comment:
I've also removed the requirement on key escaping. This is now down in the admin client itself.